### PR TITLE
Add support for detecting borsch ingredient

### PR DESCRIPTION
**Pull request by sash13**
This addition will help people test whether or not an ingredient could be used for preparing borsch.

### CODE REVIEW

**Comment by ironmaniiith** [comment]
What :sweat_smile: 
BTW instead of using it's alias, you should have used the original name `borscht` :stuck_out_tongue_winking_eye: 
(https://en.wikipedia.org/wiki/Borscht)

**Comment by jdalton** [comment]
I can't really talk as the author a lib with `kebabCase` and `castArray` :smiley: 
As with other requests we'll keep an eye on [the popularity of the request](https://github.com/arasatasaygin/is.js/blob/master/CONTRIBUTING.md#feature-requests).

**Comment by jdalton** [comment]
BTW there's no need to rebuild the minified file for the PR.

**Comment by ironmaniiith** [comment]
@jdalton , you added `enhancement` tag, wish I could give you two upvotes for that :smile:

File: test/test.js [hunk 1]
```diff
@@ -3196,4 +3196,24 @@ describe('array checks', function() {
...
+            expect(is.borschIngredient(val)).to.be.false;
+        });
+    });
+    describe('is.not.borschIngredient', function()  {
+        it('should return false if the ingredient is in the borsch', function() {
+            var val = 'beet';
+            expect(is.not.borschIngredient(val)).to.be.false;
+        });
+        it('should return true if the ingredient is not in the borsch', function() {
+            var val = 'raspberry';
```
Comment (ironmaniiith) [inline test/test.js]: @sash13 , dude could you please write a test for `arduino` as well :smile:

**Comment by mvasilkov** [comment]
Without this functionality the library is literally unusable.

**Pull request closed without merging**